tests/helpers.py

```diff
@@ -214,9 +232,11 @@ def create_pvc(sc_name, pvc_name=None):
     Returns:
         OCS: An OCS instance for the PVC
     """
-    pvc_data = copy.deepcopy(defaults.CSI_PVC_DICT)
-    pvc_data['metadata']['name'] = pvc_name if pvc_name else create_unique_resource_name(
-        'test', 'pvc'
+    pvc_data = templating.get_crd_dict(defaults.CSI_PVC_DICT)
```

Comment:
I think you can call templating.get_crd_dict(..) from ocs.defaults.py dicts definitions

Reply:
Those are no more dict definitions - just a path to the yaml

Reply:
I meant you should do this in ocs.defaults.py:
```python
CSI_PVC_DICT = templating.get_crd_dict(
    os.path.join(
        constants.TEMPLATE_PV_PVC_DIR, "PersistentVolumeClaim.yaml"
    )
)
```

In here: 
```python
pvc_data = defaults.CSI_PVC_DICT
```

Reply:
This will not work as no one is actually calling templating.get_crd_dict()
I'm just using the CSI_PVC_DICT which is never initialized.
I have to call get_crd_dict in order to get a new instance of dict everytime

Reply:
agree, that why the issue and PR was raised. We need to get rid of mutable structures from defaults.py to prevent ppl are changing them from their tests.